lib/taskana-core/src/test/java/pro/taskana/impl/WorkbasketServiceImplTest.java

```diff
@@ -335,8 +336,17 @@ public void testCreateWorkbasket_WithoutDistibutionTargets()
         verifyNoMoreInteractions(taskQueryMock, taskServiceMock, workbasketMapperMock, workbasketAccessMapperMock,
             distributionTargetMapperMock,
             taskanaEngineImplMock, taskanaEngineConfigurationMock);
-        assertThat(actualWb.getId(), not(equalTo(null)));
-        assertThat(actualWb.getId(), startsWith("WBI"));
+        assertThat(actualWbSummary.getId(), not(equalTo(null)));
+        assertThat(actualWbSummary.getId(), startsWith("WBI"));
+
+        doNothing().when(cutSpy).checkAuthorization(any(), any());
```

Comment:
A unit-test should only test 1 call per method. ALso there are few unused lines which should be removed or tested, too.